### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by jaraco** [pypa/setuptools-scm#23]
"pip install setuptools_scm" fails with this error:

```
#!bash

Downloading/unpacking setuptools-scm
  Downloading setuptools-scm-1.1.0.tar.gz
  Running setup.py egg_info for package setuptools-scm
    Traceback (most recent call last):
      File "<string>", line 14, in <module>
      File "/home/vagrant/build/setuptools-scm/setup.py", line 86, in <module>
        setuptools.setup(**arguments)
      File "/usr/lib/python2.7/distutils/core.py", line 112, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 225, in __init__
        _Distribution.__init__(self,attrs)
      File "/usr/lib/python2.7/distutils/dist.py", line 287, in __init__
        self.finalize_options()
      File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 258, in finalize_options
        ep.load()(self, ep.name, value)
      File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1989, in load
        entry = __import__(self.module_name, globals(),globals(), ['__name__'])
      File "setuptools_scm/__init__.py", line 10, in <module>
        from .version import format_version
      File "setuptools_scm/version.py", line 5, in <module>
        from pkg_resources import parse_version, SetuptoolsVersion, iter_entry_points
    ImportError: cannot import name SetuptoolsVersion
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 14, in <module>

  File "/home/vagrant/build/setuptools-scm/setup.py", line 86, in <module>

    setuptools.setup(**arguments)

  File "/usr/lib/python2.7/distutils/core.py", line 112, in setup

    _setup_distribution = dist = klass(attrs)

  File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 225, in __init__

    _Distribution.__init__(self,attrs)

  File "/usr/lib/python2.7/distutils/dist.py", line 287, in __init__

    self.finalize_options()

  File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 258, in finalize_options

    ep.load()(self, ep.name, value)

  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1989, in load

    entry = __import__(self.module_name, globals(),globals(), ['__name__'])

  File "setuptools_scm/__init__.py", line 10, in <module>

    from .version import format_version

  File "setuptools_scm/version.py", line 5, in <module>

    from pkg_resources import parse_version, SetuptoolsVersion, iter_entry_points

ImportError: cannot import name SetuptoolsVersion

----------------------------------------
Command python setup.py egg_info failed with error code 1
Storing complete log in /home/vagrant/.pip/pip.log
```

---
- Bitbucket: https://bitbucket.org/pypa/setuptools_scm/issue/23
- Originally reported by: Anonymous
- Originally created at: 2015-02-25T17:48:12.938

**Comment by jaraco** [comment]
Forgot to login when submitting it. Commenting for traceability.

---

Original comment by: Michael Hart

**Comment by jaraco** [comment]
support working with setuptools pre-12, fixes #20 #21 and #23

→ <<cset 4e74d27d5ca0>>

---

Original comment by: Ronny Pfannschmidt
